bower_components/jquery/.bower.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
I think we may have some problems:

This is what the console says when I try to run a test:

(X) GET http://localhost:8080/bower_components/jquery/dist/jquery.js 
(X) ui.js:27 Uncaught ReferenceError: $ is not defined

Did I reinstall bower incorrectly, and disconnect JQuery?